node-js-basics-101/node_modules/engine.io-parser/lib/index.js

```diff
@@ -0,0 +1,480 @@
+/**
+ * Module dependencies.
+ */
+
+var utf8 = require('./utf8');
+var hasBinary = require('has-binary2');
+var after = require('after');
+var keys = require('./keys');
+
+/**
+ * Current protocol version.
+ */
+exports.protocol = 3;
+
+/**
+ * Packet types.
+ */
+
+var packets = exports.packets = {
+    open:     0    // non-ws
+  , close:    1    // non-ws
+  , ping:     2
+  , pong:     3
+  , message:  4
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->